server/configmodule.py

```diff
@@ -119,7 +120,6 @@ class LocalBaseConfig(Config):
   AI_CONFIG_PATH = os.path.abspath(
       os.path.join(os.path.curdir, '..', 'deploy/overlays/local/ai.yaml'))
   SCHEME = 'http'
-  GCS_BUCKET = 'datcom-website-autopush-resources'
```

Comment:
autopush also sets "GCS_BUCKET", is that needed?

Reply:
yes, we are also allowing disaster dashboard on autopush, so autopush will also be reading disaster data files from GCS bucket on server start up

Reply:
I see, so the files are saved in different buckets? If this is for development purpose and all instance are synced, I wonder if it's easy to save the data in a public bucket and accessible to any instance with dashboard.

Reply:
I've saved the same files in both buckets .. but this is also temporary & will go away once we move to using the event cache API